src/plugins/intel_gpu/src/runtime/ocl/ocl_memory.hpp

```diff
@@ -39,6 +39,9 @@ struct gpu_buffer : public lockable_gpu_mem, public memory {
         assert(0 == _lock_count);
         return _buffer;
     }
+    void* buffer_ptr() const override {
```

Comment:
I believe it's not needed?

Reply:
Hi Sergey, it's not needed, this API just used in debug messages here: https://github.com/openvinotoolkit/openvino/blob/8a7c9742c2758d3806f9d23e019eaa88ef5136b1/src/plugins/intel_gpu/src/plugin/sync_infer_request.cpp#L996 
The ptr would be 0 without this API, so I added it.
such as: `GPU_Debug: sync_infer_request.cpp:989:prepare_output: result:Result_51154 with index 9 prepare output: 0`